core/templates/dev/head/pages/skill_player/ConceptCardDirective.js

```diff
@@ -21,21 +21,35 @@ oppia.directive('conceptCard', [
       restrict: 'E',
       scope: {
         returnToExploration: '&',
+        getConceptCard: '&conceptCard',
```

Comment:
I'm confused about this. Why pass it in? Better to have less coupling IMO.

Also, if the concept card is basically meant to be the same as the exploration card but with different content, it might be better to have a generic CardDirective instead. You can then pass in card-type, card-contents etc. and it can populate its content based solely on the input args.

Reply:
Removed this directive.